src/components/Avatar.js

```diff
@@ -59,7 +59,8 @@ class Avatar extends PureComponent {
                     ? (
                         <Icon
                             src={this.state.imageError ? Expensicons.FallbackAvatar : this.props.source}
-                            fill={this.state.imageError ? themeColors.offline : this.props.fill}
+                            // eslint-disable-next-line react/jsx-props-no-spreading
+                            {...(!this.state.imageError ? {fill: this.props.fill} : {})}
```

Comment:
Why this is messy, can't we simplify it?

Reply:
should we have to pass fill for the fallback avatar?

Reply:
if  getting error then not passing the fill prop otherwise fill will be pass

Reply:
@Santhosh-Sellavel 
```
                           // eslint-disable-next-line react/jsx-props-no-spreading
                            {...(!this.state.imageError ? {fill: this.props.fill} : {})}
```
these lines were added for conditional fill prop for Icon component, 
Please let me know that do we have to remove the fill prop for the fallback avatar? then I need to add a condition for the Icon component also, here default file is themeColor.icon
https://github.com/Expensify/App/blob/5f6d8db1fcd7a09371c5629bf80a16a348ec078e/src/components/Icon/index.js#L47-L51
https://github.com/Expensify/App/blob/5f6d8db1fcd7a09371c5629bf80a16a348ec078e/src/components/Icon/index.js#L33